hardware_enablement/kmm-kernel-module-management.adoc

```diff
@@ -16,6 +16,35 @@ include::modules/kmm-installation.adoc[leveloffset=+1]
 include::modules/kmm-installing-using-web-console.adoc[leveloffset=+2]
 include::modules/kmm-installing-using-cli.adoc[leveloffset=+2]
 include::modules/kmm-installing-older-versions.adoc[leveloffset=+2]
+include::modules/kmm-deploying-modules.adoc[leveloffset=+1]
+include::modules/kmm-creating-module-cr.adoc[leveloffset=+2]
+include::modules/kmm-security.adoc[leveloffset=+2]
+.Additional resources
+For more information on the synchronization mechanism, see xref:https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html[ Understanding and managing pod security admission].
+
+include::modules/kmm-example-module-cr.adoc[leveloffset=+2]
+include::modules/kmm-creating-moduleloader-image.adoc[leveloffset=+1]
+include::modules/kmm-running-depmod.adoc[leveloffset=+2]
+.Additional resources
+For more information on the Driver Tool Kit (DTK), see xref:https://docs.openshift.com/container-platform/4.12/hardware_enablement/psap-driver-toolkit.html[Driver Tool Kit (DTK)].
+
+include::modules/kmm-building-in-cluster.adoc[leveloffset=+2]
+.Additional resources
+For more information on `Build resources, see xref:https://docs.openshift.com/container-platform/4.12/cicd/builds/build-configuration.html[Build configuration resources].
+
+include::modules/kmm-using-driver-toolkit.adoc[leveloffset=+2]
+.Additional resources
+For more information on the Driver Toolkit, see xref:https://docs.openshift.com/container-platform/4.12/hardware_enablement/psap-driver-toolkit.html[Driver Toolkit].
```

Comment:
xrefs need to be relative (I am guessing at one `../` but it might be `../../` so check me in the output :slightly_smiling_face:), and additional resources are just the bare link with bullets but no surrounding text 
```suggestion
.Additional resources
* xref:../authentication/understanding-and-managing-pod-security-admission.html[ Understanding and managing pod security admission]

include::modules/kmm-example-module-cr.adoc[leveloffset=+2]
include::modules/kmm-creating-moduleloader-image.adoc[leveloffset=+1]
include::modules/kmm-running-depmod.adoc[leveloffset=+2]
.Additional resources
* xref:../hardware_enablement/psap-driver-toolkit.html[Driver Tool Kit (DTK)]

include::modules/kmm-building-in-cluster.adoc[leveloffset=+2]
.Additional resources
* xref:../cicd/builds/build-configuration.html[Build configuration resources]

include::modules/kmm-using-driver-toolkit.adoc[leveloffset=+2]
.Additional resources
* xref:../hardware_enablement/psap-driver-toolkit.html[Driver Toolkit]
```

Reply:
@jeana-redhat Oh. I'd always done it the other way using the URL. I'll change them.